.github/workflows/build.yml

```diff
@@ -27,6 +27,7 @@ jobs:
         with:
           args: release --snapshot --skip-publish --rm-dist
       - name: Scan image for vulnerabilities
-        uses: docker://docker.io/aquasec/trivy:0.2.1
+        uses: docker://docker.io/aquasec/trivy:0.4.0
+        continue-on-error: true
```

Comment:
You can specify the severity level at which you want Trivy to fail - maybe this would be best off not continuing on error, but only failing on high (or critical) vulnerabilities

Reply:
Added `--exit-code 1 --severity CRITICAL`